include/os/linux/spl/sys/isa_defs.h

```diff
@@ -223,10 +223,34 @@
 
 #define	_ALIGNMENT_REQUIRED	1
 
+/*
+ * loongarch arch specific defines
+ * only loongarch64 is supported yet
+ */
+
+#elif defined(__loongarch__)
+
+#if !defined(__loongarch64)
+#define	__loongarch64
+#endif
+
+#define	_ZFS_LITTLE_ENDIAN
+#define	_SUNOS_VTOC_16
+
+#ifndef _LP64
+#define	_LP64
+#endif
```

Comment:
better check for `__loongarch_lp64`, as the `__loongarch64` macro [is deprecated](https://loongson.github.io/LoongArch-Documentation/LoongArch-toolchain-conventions-EN.html). You could fold the LP64 check into the `__loongarch__` check just like the riscv case.